src/main/java/com/greenfox/tribes1/Building/BuildingService.java

```diff
@@ -0,0 +1,27 @@
+package com.greenfox.tribes1.Building;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BuildingService {
+
+  private BuildingRepository buildingRepository;
+
+  @Autowired
+  public BuildingService(BuildingRepository buildingRepository) {
+    this.buildingRepository = buildingRepository;
+  }
+
+  public boolean isValidBuilding(Building building){
+    return building!=null;
+  }
+
+  public Building saveBuilding(Building building){
```

Comment:
Missing spaces after methods, and at the if statement.
Should look like:

if () {
}